### PR TITLE
Add light client, github and theme change buttons in mobile view

### LINKED ISSUE #113: Add Sidebar icons to mobile version
At the moment mobile sidebar is missing the github/lightclient/theme icons.

### PR DESCRIPTION
**Pull request by wirednkod**
closes #113 

---

Submission checklist:

#### Layout

Add following missing icons to the mobile sidebar:
- [x] Add Light client loader (when network selected) 
- [x] Add github link
- [x] Add theme selector

### CODE REVIEW

**Comment by Tbaut** [comment]
the build failed, can you check?

File: src/header.tsx [hunk 1]
```diff
@@ -45,10 +49,12 @@ if (import.meta.env.DEV) {
 }
 
 export const Header = () => {
-  const { network, setNetwork } = useNetwork()
+  const { network, setNetwork, lightClientLoaded, isLight } = useNetwork()
   const { accounts, selectAccount, selectedAccount } = useAccounts()
   const [, disconnectAll] = useWalletDisconnector()
   const [isConnectionDialiogOpen, setIsConnectionDialiogOpen] = useState(false)
+  // eslint-disable-next-line
```
Comment (Tbaut) [inline src/header.tsx:56]: Any clue why we need that here, and not everywhere else (I saw it once otherwise)

File: src/header.tsx [hunk 2]
```diff
@@ -69,16 +75,48 @@ export const Header = () => {
...
+                  {!lightClientLoaded ? (
+                    <TbLoaderQuarter className="h-5 w-5 animate-spin" />
+                  ) : (
+                    <FaCheckCircle className="text-[#00b300]" />
+                  )}
+                  Light Client {!lightClientLoaded ? `syncing` : `synced`}
+                </a>
+              )}
+              <a
+                href="https://github.com/polkadot-fellows"
```
Comment (Tbaut) [inline src/header.tsx:103]:
```suggestion
                href="https://github.com/delegit-xyz/dashboard"
```

**Review by Tbaut — APPROVED** [review]
Good to go with the right link :)

File: src/header.tsx (hunk 1 above)
Comment (wirednkod) [reply src/header.tsx:56]: It mentions that the `useTheme` for some reason needs a callback but I cannot understand why 🤷🏽

Comment (Tbaut) [reply src/header.tsx:56]: yeah I saw the error, asking why since it wasn't clear to me. No big issue, we can close our eyes for now.

**Pull request merged**